scripts/cluster-init.sh

```diff
@@ -27,6 +27,10 @@ kubectl apply -f deploy/neonvm.yaml | indent
 
 kubectl wait deployment -n neonvm-system neonvm-controller --for=condition=Available=True | indent
 
+for img in pg14-disk-test:dev kube-autoscale-scheduler:dev autoscaler-agent:dev; do
```

Comment:
```suggestion
for img in kube-autoscale-scheduler:dev autoscaler-agent:dev; do
```

(note: this requires changing the README to do the appropriate `kind load` there)

The test VM image isn't required for cluster startup, so I'd rather not require it to exist if we don't need to.

That said, [see note about switching away from these scripts].